lib/cog/repository/bundles.ex

```diff
@@ -292,6 +299,30 @@ defmodule Cog.Repository.Bundles do
   end
 
   @doc """
+  Disable old incompatible bundles
+  """
+  def disable_incompatible_bundles() do
+    query =
```

Comment:
How many transactions does this function require assuming `n` incompatible bundles?